third_party/tensorflow_patches/fix_armhf_xnnpack.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/.bazelrc b/.bazelrc
+index b1d4eee905d..ada1395298f 100644
+--- a/.bazelrc
++++ b/.bazelrc
+@@ -572,6 +572,7 @@ build:elinux_aarch64 --distinct_host_configuration=true
+ build:elinux_armhf --config=elinux
+ build:elinux_armhf --cpu=armhf
+ build:elinux_armhf --distinct_host_configuration=true
++build:elinux_armhf --copt -mfp16-format=ieee
```

Comment:
This is from https://github.com/tensorflow/tensorflow/commit/4ec84aa99372ca7899fb12715edd2bfe3c947c88